CMakeLists.txt

```diff
@@ -186,10 +184,7 @@ if(BUILD_DISTRIBUTABLE_LIBRARY)
   set(OPENSSL_USE_STATIC_LIBS TRUE)
 endif()
 find_package(OpenSSL REQUIRED)
-
-if(ENABLE_LIBCURL)
-  find_package(CURL REQUIRED)
-endif()
+find_package(CURL REQUIRED)
```

Comment:
Just ensure that libcurl is statically linked in when the distributable jar is uploaded to Maven central. Some GATK users were on a machine which didn't have libcurl installed.

Reply:
Agreed. Looks like all curl methods are fully resolved in the distributable libtiledbgenomicsdb.so -
```
[vagrant@localhost lib]$ objdump -t -T libtiledbgenomicsdb.so | grep -i " curl_" | grep easy
0000000000917000 g     F .text	00000000000001c3              curl_easy_perform
0000000000917240 g     F .text	0000000000000031              curl_easy_getinfo
0000000000917950 g     F .text	00000000000000ad              curl_easy_send
0000000000917600 g     F .text	0000000000000149              curl_easy_reset
```
Also, see this - https://github.com/GenomicsDB/GenomicsDB/blob/e18fa6378214579c75811616d449a7d3a3e77dc5/cmake/Modules/FindCURL.cmake#L37.